pkg/neorpc/filters.go

```diff
@@ -134,6 +152,16 @@ func (f NotificationFilter) IsValid() error {
 	if f.Name != nil && len(*f.Name) > runtime.MaxEventNameLen {
 		return fmt.Errorf("%w: NotificationFilter name parameter must be less than %d", ErrInvalidSubscriptionFilter, runtime.MaxEventNameLen)
 	}
+	if len(f.Parameters) != 0 { // todo: limit max size? what number?
+		for i, parameter := range f.Parameters {
+			if parameter.Type < smartcontract.AnyType || parameter.Type > smartcontract.SignatureType {
```

Comment:
not sure i think it is buggy, because allowed types are bounded above and i _really do not expect_ these enums are ever changed (expansion is ok). but don't mind